code/game/objects/structures/morgue.dm

```diff
@@ -29,10 +31,18 @@
 	if(connected)
 		icon_state = "morgue0"
 	else
-		if(contents.len)
-			icon_state = "morgue2"
-		else
-			icon_state = "morgue1"
+		icon_state = "morgue1"
+		var/list/searching = GetAllContents(searchDepth = 3, checkClient = 0) // Search inside bodybags as well.
+		for(var/mob/living/M in searching)
+			if(iscarbon(M))
+				var/mob/living/carbon/C = M
+				switch(C.stat)
+					if(DEAD)
+						icon_state = "morgue2"
+					if(UNCONSCIOUS)
+						icon_state = "morgue3"
+					if(CONSCIOUS)
+						icon_state = "morgue4"
```

Comment:
you can either break here after finding the first mob and setting the state, or you can save the stats. meaning if there's three dead, but one alive, you show the alive one

Reply:
also if possible you can make these names a bit more clear